pom.xml

```diff
@@ -95,6 +105,23 @@
         <artifactId>nv-i18n</artifactId>
         <version>${version.com.neovisionaries}</version>
       </dependency>
+      <!-- JSON testing -->
+      <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-test</artifactId>
+        <version>2.2.6.RELEASE</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>com.jayway.jsonpath</groupId>
```

Comment:
org.skyscreamer and json-path are required for spring-boot-test? Cannot fins any imports of them

Reply:
Those are test-runtime dependencies, so no imports indeed. Look for `org.springframework.boot.test.json.JacksonTester` imports.